Makefile

```diff
@@ -103,6 +105,10 @@ all: lint test manager
 native-test:
 	GO111MODULE=on go test -mod vendor ./pkg/... ./apis/... -race -bench . -coverprofile cover.out
 
+.PHONY: benchmark-test
+benchmark-test:
+	go test ./pkg/... -bench . -run="^#" -count 10 > ${BENCHMARK_FILE_NAME}
```

Comment:
looks like this is not used anymore?

Reply:
Yes! I kept it since it's still useful if the dev wants to run it locally.

Reply:
can we add GOMAXPROCS=1 here and use this in the github action